src/lib/telemetry.js

```diff
@@ -0,0 +1,30 @@
+const Mixpanel = require('mixpanel');
+const loadJson = require('load-json-file');
+const writeJson = require('write-json-file');
+const { cryptoRandomObjectId } = require('@apify/utilities');
+const { MIXPANEL_TOKEN, TELEMETRY_FILE_PATH } = require('./consts');
+
+const mixpanel = Mixpanel.init(MIXPANEL_TOKEN, { keepAlive: false });
+
+/**
+ * Returns telemetry distinctId for current local environment or creates new one.
+ * @returns {Promise<*|string>}
+ */
+const getOrCreateLocalDistinctId = () => {
+    try {
+        const telemetry = loadJson.sync(TELEMETRY_FILE_PATH);
+        return telemetry.distinctId;
+    } catch (e) {
```

Comment:
We should check that it's an error due to a nonexisting file and otherwise:
- report it to mixpanel
- swallow or throw? not sure, but probably swallow not to interrupt the usage?